go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/wangkuiyi/gotorch
```

Comment:
I agree with enabling Go modules, which can speed up downloading packages with [GOPROXY=https://goproxy.cn](https://goproxy.cn/) in China, the [GOPROXY](https://golang.org/ref/mod#goproxy-protocol) required `GO111MODULE=on `. Please @wangkuiyi also takes a look at this PR.